src/algebra/algebra/spectrum.lean

```diff
@@ -212,6 +213,26 @@ theorem preimage_units_mul_eq_swap_mul {a b : A} :
   (coe : Rˣ → R) ⁻¹' σ (a * b) = coe ⁻¹'  σ (b * a) :=
 by { ext, exact unit_mem_mul_iff_mem_swap_mul, }
 
+section star
+
+variables [star_add_monoid R] [star_ring A] [star_module R A]
+
+local postfix `⋆`:std.prec.max_plus := star
+
+lemma star_mem_resolvent_set {r : R} {a : A} (h : r ∈ resolvent_set R a) :
+  r⋆ ∈ resolvent_set R a⋆ :=
+by simpa only [mem_resolvent_set_iff, algebra.algebra_map_eq_smul_one, star_sub, star_smul,
+  star_one] using is_unit.star h
+
+protected lemma map_star (a : A) : (σ a)⋆ = σ a⋆ :=
```

Comment:
```suggestion
protected lemma map_star (a : A) : σ a⋆ = (σ a)⋆ :=
```
or
```suggestion
protected lemma star_map (a : A) : (σ a)⋆ = σ a⋆ :=
```

Reply:
? In my head we are mapping the star operation over elements of the spectrum, so `map_star` makes sense to me the way I have it.

Reply:
`foo.map_add` means `foo (x + y) = foo x + foo y`. This lemma is called `spectrum.map_star`, so it should be about what applying `spectrum` to `star` does.

Reply:
done